cmd/agola/cmd/runlist.go

```diff
@@ -65,6 +65,21 @@ func printRuns(runs []*gwapitypes.RunResponse) {
 		fmt.Printf("%s: Phase: %s, Result: %s\n", run.ID, run.Phase, run.Result)
 		for _, task := range run.Tasks {
 			fmt.Printf("\tTaskName: %s, Status: %s\n", task.Name, task.Status)
+			printTaskDetails(run.ID, task.ID)
+		}
+	}
+}
+
+func printTaskDetails(runid, taskid string) {
+	gwclient := gwclient.NewClient(gatewayURL, token)
+	task, _, err := gwclient.GetRunTask(context.TODO(), runid, taskid)
```

Comment:
I think we should first collect all the information and then print them or the output will appear task by task on connections with high latency.